lib/defaultFormatter.js

```diff
@@ -0,0 +1,45 @@
+var chalk = require('chalk');
+var path = require('path');
+
+module.exports = function() {
+  return function(input) {
```

Comment:
We're using chalk, so the user should already be able to specify `--no-color` :smile:

https://github.com/sindresorhus/chalk#chalksupportscolor

If you plan to add options to it then it's not a big deal. :+1: